Makefile

```diff
@@ -27,6 +27,20 @@ assets:
 	python manage.py assets build ${TAIL}
 	chmod 664 ${ASSETS_DIR}.webassets-cache/*
 
+travis-assets:
+	if [ -d "${ASSETS_DIR}.webassets-cache/" ]; then \
+		echo "eating cache - yum!"; \
+	else \
+		cd ${JS_DIR} && \
+		npm install && \
+		cd ${CWD}; \
+		python manage.py webpack --extra=--display-error-details --extra=--dev --extra=--nowatch; \
```

Comment:
As already mention, this has to read `python manage.py webpack --dev --nowatch --extra=--display-error-details`